docs/lib/analytics/identifyuser.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Identify user
-description: Use the Amplify analytics plugin to inform Pinpoint about your users. This could be an unauthenticated or an authenticated user.
+description: Use the Amplify analytics plugin to inform Pinpoint about your users. This could be an authenticated user.
```

Comment:
just `This could be an authenticated user.` doesn't really make sense here, can just remove entire sentence

Reply:
Fixed